docs/Background-Mode.md

```diff
@@ -9,27 +9,29 @@ redirect_from:
 
 # Background Mode
 
+React Native Track Player supports playing audio while your app is in the background on all supported platforms.
+
 ## Android
-The background support works right out of the box, and it will keep playing even after the app is closed. If that is not the desired behavior and you only want to play when the app is open, you can disable it with the `stopWithApp` property in `updateOptions`:
+Background audio playback works right out of the box. By default, the audio will keep playing even after the app is closed. If that is not the desired behavior and you want the audio to stop playing when the app is closed, you can disable it with the `stopWithApp` property in `updateOptions`:
```

Comment:
Can we word this in a way that more clearly states the stopWithApp has to do with when the app is closed/quit, which is different than when it is suspended for multitasking?

Reply:
Hmm - would you like to make a proposal for this?

Reply:
Totally not related to this PR, but I've been thinking about making `stopWithApp` false by default in v2, as it's closest to how iOS behaves and the majority of users also set this to false.
As both iOS and Windows need extra steps for background mode to work, I think it would be more intuitive if Android worked the same way.

Reply:
Here is my stab at it:

> Background audio playback works right out of the box. By default, the audio will continue to play, not only when the app is suspended in the background, but also after the app is closed by the user. If that is not the desired behavior and you want the audio to stop playing when the app is closed, you can disable it with the `stopWithApp` property in `updateOptions`. In this case, the audio will still play while the app is open in the background.

Reply:
Perfect - will add it.